src/PrestaShopBundle/Command/ModuleCommand.php

```diff
@@ -81,7 +81,7 @@ protected function init(InputInterface $input, OutputInterface $output)
         $this->translator = $this->getContainer()->get('translator');
         $this->input = $input;
         $this->output = $output;
-        require $this->getContainer()->get('kernel')->getRootDir() . '/../config/config.inc.php';
+        require(_PS_CONFIG_DIR_ . 'config.inc.php');
```

Comment:
You must run php-cs-fixer. You add parenthesis here, but require is not a function :) (even if it's work)

Reply:
![csfixer](https://user-images.githubusercontent.com/26626066/49643569-5a89b980-fa16-11e8-8f14-d4e9c38cdd5d.png)
I can use PHP-CS-Fixer manually from it FriendsOfPHP with `php-cs-fixer.phar` , but I did never figure out how to use `php ./vendor/bin/php-cs-fixer fix` from Prestashop. I am not windows. I did install composer. Can you learn me how to do that? For a lot of files, it is not easy to do it manually.

I attached a screenshot.

Reply:
I really don't know how Windows and its ecosystem works :/ Maybe try to call the script directly: ./vendor/friendsofphp/php-cs-fixer/php-cs-fixer ?

Reply:
`./vendor/friendsofphp/php-cs-fixer/php-cs-fixer` worked! Thank you.